docs/content/en/working_with_findings/finding_priority.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Finding Priority Enhancement (Pro)"
+description: "How DefectDojo ranks your Findings"
+weight: 1
+---
+
+Additional Finding filters are available in DefectDojo Pro to more easily triage, filter and prioritize Findings.
+
+* Priority sorts Findings based on the context and importance of the Product they are stored in.
+* Risk considers the Product's context, with a greater emphasis on the exploitability of a Finding.
+
+## Finding Priority
+
+In DefectDojo Pro, Priority is a calculated field on Findings that can be used to sort or filter Findings according to Product-level metadata:
+
+- Product's Business Criticality
+- Whether the Product has an External Audience
+- Whether the Product is Internet Accessible
+- The Product's estimated revenue or user records count 
+
+DefectDojo Pro's Finding Priority assigns a numerical rank to each Finding according to this metadata, to provide users with a stronger context on triage and remediation.
+
+![image](images/pro_finding_priority.png)
+
+The range of Priority values is from 0 to 1150.  The higher the number, the more urgency the Finding is to triage or remediate.
+
+Priority numbers can be used with other filters to compare Findings in any context, such as:
+
+* within a single Product, Engagement or Test
+* globally in all DefectDojo Products
+* between a few specific Products
+
+## How Priority is calculated
+
+Every Active finding will have a Priority calculated.  Inactive or Duplicate Findings will not.
+
+Priority is set based on the following factors:
+
+#### Product-Level
+
+- The assigned Criticality for the Product (if defined)
+- The estimated User Records for the Product (if defined)
+- The estimated Revenue for the Product (if defined)
+- If the Product has External Audience defined
+- If the Product has Internet Accessible defined.
+
+All of these metadata fields can be set on the Edit Product form for a given Product.
+
+#### Finding-Level
+
+- Whether or not the Finding has an [EPSS score](/en/working_with_findings/intro_to_findings/#monitor-current-vulnerabilities-using-cves-and-epss-scores-pro-feature), this is automatically kept up to date for Pro customers
+- How many Endpoints in the Product are affected by this Finding
+- Whether or not a Finding is Under Review
+
+If Product-level metadata is not set, the Priority level will follow the Severity for a given Finding:
```

Comment:
I hate to be 'that guy' but actually those severity to priority mappings only occur when there is NO metadata at the Product or Findings level.

You'd have to have a Finding with no endpoints, no Product-level meta data, no EPSS and not "Under Review", etc - only Finding severity exists to drive priority. That "base priority" is a fail-safe to cover this situation.

This shouldn't happen that often but it is possible.

So I'd change that to:

> If no metadata is set at the Product or Finding level, the Priority level will follow the Severity for a given Finding: